org.uqbar.project.wollok.tests/src/org/uqbar/project/wollok/tests/interpreter/CollectionTestCase.xtend

```diff
@@ -295,5 +295,18 @@ class CollectionTestCase extends AbstractWollokInterpreterTestCase {
 			assert.equals(34, numbers.sum {n => n})
 		}'''.interpretPropagatingErrors
 	}
+
+	@Test
+	def void concatenation() {
+		'''
+		program p {
+			«instantiateCollectionAsNumbersVariable»
```

Comment:
This line is not needed. This was because all test methods on where using the same collection as "input". 
Instead of writing it on each test I was calling that method to generate the code.
Also because there were many test methods that were exactly the same for List and Sets, the only difference was the initialization of the collection (list or set literal). So in order to make this tests shareable in a super class I needed to introduce this indirection.

In your case you are using other lists declared in the same text of the program, so you don't need this line